pandera/schema_components.py

```diff
@@ -29,15 +29,15 @@ class Column(SeriesSchemaBase):
     has_subcomponents = False
 
     def __init__(
-        self,
-        pandas_dtype: PandasDtypeInputTypes = None,
-        checks: CheckList = None,
-        nullable: bool = False,
-        allow_duplicates: bool = True,
-        coerce: bool = False,
-        required: bool = True,
-        name: str = None,
-        regex: bool = False,
+            self,
```

Comment:
the formatting changes on here will break the linter (pandera uses `black` to lint/auto-format the code), can you revert these changes?